app/templating/summary/formatters/abstract_formater.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABCMeta, abstractmethod
+
+
+class AbstractFormatter(metaclass=ABCMeta):
+
+    @staticmethod
+    @abstractmethod
+    def format(schema_answers, state_answers, user_answer):
```

Comment:
As discussed, this doesn't really lend itself at the moment, with the updated questionnaire state model this should improve